tests/TestHarness/launcher.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import pathlib
+import sys
+
+from testUtils import Utils
+
+class launcher(object):
+    def parseArgs(self, args):
+        '''Configure argument parser and use it on the passed in list of strings.
+        
+        arguments:
+        args -- list of arguments (may be sys.argv[1:] or synthetic list)
+        
+        returns -- argparse.Namespace object with parsed results
+        '''
+        def comma_separated(string):
+            return string.split(',')
+        parser = argparse.ArgumentParser(description='launcher command line options',
+                                         formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+        parser.add_argument('-i', '--timestamp', help='set the timestamp for the first block.  Use "now" to indicate the current time')
+        parser.add_argument('-l', '--launch', choices=['all', 'none', 'local'], help='select a subset of nodes to launch.  If not set, the default is to launch all unless an output file is named, in which case none are started.', default='all')
+        parser.add_argument('-o', '--output', help='save a copy of the generated topology in this file', dest='topology_filename')
+        parser.add_argument('-k', '--kill', help='retrieve the list of previously started process ids and issue a kill to each')
+        parser.add_argument('--down', type=comma_separated, help='comma-separated list of node numbers that will be shut down')
```

Comment:
Looks like the following is missing:
`    ("bounce", bpo::value<string>(&bounce_nodes),"comma-separated list of node numbers that will be restarted using the eosio-tn_bounce.sh script")
`

Reply:
Looks like this was addressed in https://github.com/AntelopeIO/leap/pull/331/commits/5ec19e70dcab8d86971a932987cdf31155bd2f7b